features/guidance/notification.feature

```diff
@@ -0,0 +1,65 @@
+@routing  @guidance @mode-change
+Feature: Notification on turn onto mode change
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 20 meters
+
+    Scenario: Turn onto a Ferry
+        Given the node map
+            |   |   |   |   |   | f |
```

Comment:
f is not used here, right? I guess since you didn't see any notification on the exit? But in that case it would be alright, since the turn is not obvious (left/right are more or less equal)